src/ducks/appsSuggestions/AppsSuggestion.js

```diff
@@ -0,0 +1,45 @@
+import { Document } from 'cozy-doctypes'
+import { head } from 'lodash'
+import { TRANSACTION_DOCTYPE } from 'doctypes'
+
+class AppsSuggestion extends Document {
+  static fetchBySlug(slug) {
+    return this.queryAll({ slug }).then(head)
+  }
+
+  static linkTransaction(suggestion, transaction) {
```

Comment:
I don't know why but it seems weird to have `link` here where the original suggestion is not modified.